addon/errors.js

```diff
@@ -4,12 +4,13 @@ const {
   A: emberArray,
   Object: EmberObject,
   get,
-  set
+  set,
+  run
 } = Ember;
 
 export default EmberObject.extend({
   unknownProperty(property) {
-    set(this, property, emberArray());
-    return get(this, property);
+    run.once(() => { set(this, property, emberArray()); });
```

Comment:
In that case I'm unsure how to resolve this. At least as a temp solution moving the `set` into the runloop allows to avoid getting the hard error `You modified errors.model twice in a single render` from #438 and is also the often-mentioned solution from the [original discussion of the Glimmer 2 issue](https://github.com/emberjs/ember.js/issues/13948).

Interestingly, the error is only thrown on validation of existing records.

If needed, I can try to reproduce this in a Twiddle.